plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/totals/shipping/index.tsx

```diff
@@ -137,6 +138,7 @@ export const TotalsShipping = ( {
 			{ showShippingCalculatorForm && (
 				<ShippingCalculator
 					onUpdate={ () => {
+						dispatchStoreEvent( 'calculate-shipping' );
```

Comment:
@senadir what about this event? Should we use `dispatchCheckoutEvent` here as well?